### PR TITLE
Add incoming PRs to the API response

### PR DESCRIPTION
**Pull request by samuelli**

### CODE REVIEW

**Comment by samuelli** [comment]
Test failure is blocked on recording bug since we refactored to have the build/ dir

File: src/server/dash-server.ts [hunk 1]
```diff
@@ -185,15 +182,8 @@ export class DashServer {
...
-              author: '',
-              createdAt: Date.parse(review.createdAt),
-              reviewState: review.state,
-            };
-            if (review.author && review.author.__typename === 'User') {
-              result.author = review.author.login;
-            }
-            object.reviews.push(result);
+
+            object.reviews.push(reviewFieldsToResult(review));
```
Comment (gauntface) [inline src/server/dash-server.ts]: Could you name object to something more descriptive.

File: src/server/dash-server.ts [hunk 2]
```diff
@@ -204,9 +194,61 @@ export class DashServer {
...
+
+      // Incoming reviews
+      for (const pr of result.data.incomingReviews.nodes || []) {
+        if (!pr || pr.__typename !== 'PullRequest') {
+          continue;
+        }
+
+        const object: IncomingPullRequest = {
+          ...prFieldsToResult(pr),
+          myReview: null,
```
Comment (gauntface) [inline src/server/dash-server.ts]: What is this used for?

File: src/server/dash-server.ts (hunk 1 above)
Comment (samuelli) [reply src/server/dash-server.ts]: Done.

File: src/server/dash-server.ts (hunk 2 above)
Comment (samuelli) [reply src/server/dash-server.ts]: This fields contains your latest review.

**Pull request merged**